storybook/storybook-react/src/components/Switch/Switch.stories.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Switch } from '@amsterdam/design-system-react/src'
+import { Meta, StoryObj } from '@storybook/react'
+
+import '@amsterdam/design-system-css/src/switch/switch.scss'
+
+const meta = {
+  title: 'Switch',
+  component: Switch,
+} satisfies Meta<typeof Switch>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    disabled: false,
+  },
+}
```

Comment:
Let's show the 2 cases described in `packages/react/src/Switch/README.md` in Storybook, so the default can be a Switch with `aria-label` and another Story can be the Switch with a FormLabel. That way, the examples in storybook show the best practice

Reply:
I guess you mean something like this?
```
   <label htmlFor="switch-label">
        Switch
        <Switch {...args} id="switch-label" />
      </label>
```

how would you style the `label`? I would assume to do this with css modules. But this give me a typescript error for unknown module. Styling is applied, but we need to update the tsconfig. Are there any other options?

Reply:
No we can't, currently the Switch has a label enclosed to allow the click on a hidden input element. I would prefer to always wrap the input in a label but I recall you wanted to allow a switch without a label?

Reply:
Your switch component needs to look like this:
```
<label for="switch-1" class="amsterdam-switch">
          Default
          <input type="checkbox" role="switch" id="switch-1">
</label>
```